configs/config.d/50-stash-origin-authz.cfg

```diff
@@ -32,12 +32,22 @@ if named stash-origin-auth
 
    sec.protbind * gsi
 
-   acc.authdb /etc/xrootd/Authfile
+   if defined STASH_ORIGIN_AUTHFILE
+      acc.authdb $STASH_ORIGIN_AUTHFILE
```

Comment:
```suggestion
      acc.authdb $STASH_ORIGIN_AUTH_AUTHFILE
```

Reply:
I understand why but this makes me sad

Reply:
Can’t stomach the name. How about:

STASH_ORIGIN_PUBLIC_AUTHFILE

below and keep the one here?